server/src/main/java/org/opensearch/discovery/InitializeExtensionResponse.java

```diff
@@ -46,18 +48,22 @@
  */
 public class InitializeExtensionResponse extends TransportResponse {
     private String name;
+    private List<String> implementedInterfaces;
```

Comment:
@dbwiddis need suggestion for a better name instead of calling it `implementedInterfaces`, can we say just `modules`?

Reply:
@owaiskazi19 what terminology does the plugin ecosystem use?  

`ActionModule` is a concrete class that is different than `ActionPlugin` interface, so I'm not sure modules is right.

Reply:
`modules` was just a sample suggestion. Plugin ecosystem filters the plugin based on the class and doesn't have any name for the interfaces. Probably we can call them as `extensionServices` or `extensionInterfaces`?